data/itemGenerator.js

```diff
@@ -421,6 +459,82 @@ function generatePerson(baseUrl, golden) {
   };
 }
 
+/**
+ * @param {string} orgBaseUrl
+ * @param {boolean} golden
+ */
+function generatePlace(orgBaseUrl, golden) {
+  var siteName = faker.address.streetName() + " " + faker.random.arrayElement(["Sports Village", "Leisure Centre", "Centre"]);
+  var postcodeObj = postcodes[faker.random.number(postcodes.length - 1)];
+  return {
+    "type": "Place",
+    "url": orgBaseUrl + "/" + faker.random.arrayElement(["sites", "centres", "locations"]) + "/" + faker.helpers.slugify(siteName.toLowerCase()),
+    "name": siteName,
+    "description": faker.lorem.paragraphs(golden ? 4 : faker.random.number(4)),
+    "identifier": faker.finance.bic(),
+    "address": {
+      "type": "PostalAddress",
+      "streetAddress": faker.address.streetAddress(),
+      "addressLocality": "Oxford",
+      "addressRegion": "Oxfordshire",
+      "postalCode": postcodeObj.postcode,
+      "addressCountry": "GB"
+    },
+    "telephone": faker.phone.phoneNumber(),
+    "geo": {
+      "type": "GeoCoordinates",
+      "latitude": postcodeObj.latitude,
+      "longitude": postcodeObj.longitude
+    },
+    "image": generateImages(),
+    "amenityFeature": generateAmenityFeature(golden),
+    "openingHoursSpecification": [
+      {
+        "type": "OpeningHoursSpecification",
+        "dayOfWeek": "https://schema.org/Sunday",
+        "opens": "09:00",
+        "closes": "17:30"
+      },
+      {
+        "type": "OpeningHoursSpecification",
+        "dayOfWeek": "https://schema.org/Monday",
+        "opens": "06:30",
+        "closes": "21:30"
+      },
+      {
+        "type": "OpeningHoursSpecification",
+        "dayOfWeek": "https://schema.org/Tuesday",
+        "opens": "06:30",
+        "closes": "21:30"
+      },
+      {
+        "type": "OpeningHoursSpecification",
+        "dayOfWeek": "https://schema.org/Wednesday",
+        "opens": "06:30",
+        "closes": "21:30"
+      },
+      {
+        "type": "OpeningHoursSpecification",
+        "dayOfWeek": "https://schema.org/Thursday",
+        "opens": "06:30",
+        "closes": "21:30"
+      },
+      {
+        "type": "OpeningHoursSpecification",
+        "dayOfWeek": "https://schema.org/Friday",
+        "opens": "06:30",
+        "closes": "20:30"
+      },
+      {
+        "type": "OpeningHoursSpecification",
+        "dayOfWeek": "https://schema.org/Saturday",
+        "opens": "07:15",
+        "closes": "17:30"
+      }
+    ]
+  };
+}
+
```

Comment:
This is an exact copy of the `Place` object that was created in `generateSessionSeriesItemData`. I have made no updates to it other than moving it into its own function